### PR TITLE
gottcha2 recipe

### PR DESCRIPTION
**Pull request by poeli**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/gottcha2/2.1.4_beta/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{% set name = "GOTTCHA2" %}
+{% set version = "2.1.4_beta" %}
```
Comment (bgruening) [inline recipes/gottcha2/2.1.4_beta/meta.yaml]: Do we really need the beta version? Can we wait for a stable version.

Comment (druvus) [reply recipes/gottcha2/2.1.4_beta/meta.yaml]: @poeli would you as the upstream author consider to package GOTTCHA2 using a setup.py file? Another improvement would be to include a download function to retrieve the databases automatically.

Comment (poeli) [reply recipes/gottcha2/2.1.4_beta/meta.yaml]: @bgruening The version is quite stable -- I'm comfortable to remove the "beta".
@druvus Thanks for the suggestions. I will make one.

Comment (stain) [reply recipes/gottcha2/2.1.4_beta/meta.yaml]: I would keep this PR open until @druvus have requested/added a non-beta version upstream with the database download function

**Comment by druvus** [comment]
@bioconda-bot please update

**Pull request closed without merging**